Include page object utility in nimble-components build output (#2290)
ni/nimble: packages/nimble-components/src/utilities/testing/README.md

# Utilities Testing

> The `utilities/testing` folder contains test utilities to be used by nimble-component page objects and tests. They are exported externally since public page objects need to use them but are not part of Nimble's public API.

Reviewer: ```suggestion
The `utilities/testing` folder contains test utilities to be used by nimble-component page objects and tests. The utilities are used by public page objects but are not considered part of Nimble's public API.
```
